Use the plugdev group for bluetooth on Slackware, and allow this to
  work as expected with our current setup.

diff -Nur bluez-4.61.orig//src/bluetooth.conf bluez-4.61/src/bluetooth.conf
--- bluez-4.61.orig//src/bluetooth.conf	2010-02-18 17:48:50.434527785 -0600
+++ bluez-4.61/src/bluetooth.conf	2010-02-18 17:47:30.727431004 -0600
@@ -11,11 +11,16 @@
     <allow own="org.bluez"/>
     <allow send_destination="org.bluez"/>
     <allow send_interface="org.bluez.Agent"/>
+    <allow send_interface="org.bluez.Adapter"/>
     <allow send_interface="org.bluez.HandsfreeAgent"/>
   </policy>
 
-  <policy at_console="true">
-    <allow send_destination="org.bluez"/>
+  <policy group="plugdev">
+     <allow send_destination="org.bluez"/>
+     <allow send_interface="org.bluez.Agent"/>
+     <allow send_interface="org.bluez.Adapter"/>
+     <allow send_destination="org.bluez"
+            send_interface="org.freedesktop.DBus.Introspectable"/>
   </policy>
 
   <!-- allow users of lp group (printing subsystem) to 
